### DISCUSSION TITLE
App desktop

### DISCUSSION THREAD

**Discussion by thyagojcosta** [balta-io/forum#240] [Tira Dúvidas] [answered]
Fala, Balta!!!! Tudo bem? To precisando criar uma aplicação desktop. Desenvolvi toda lógica como um servidor no formato Web Api, com sqlite como banco local (é o suficiente) e pensei em utilizar o electron.net pra criar o app! Mas sinceramente estou muito em dúvida. Pensei em utilizar uma solução hibrida com Blazor e WF.

Gostaria de uma opinião sua e uma sugestão de stack para criar um App desktop com .NET e de fácil personalização do frontend e integração

Abraço

**Comment by thyagojcosta** [comment, marked as answer]
> Fala, Balta!!!! Tudo bem? To precisando criar uma aplicação desktop. Desenvolvi toda lógica como um servidor no formato Web Api, com sqlite como banco local (é o suficiente) e pensei em utilizar o electron.net pra criar o app! Mas sinceramente estou muito em dúvida. Pensei em utilizar uma solução hibrida com Blazor e WF.
> 
> Gostaria de uma opinião sua e uma sugestão de stack para criar um App desktop com .NET e de fácil personalização e integração
> 
> Abraço
> 
> 

@andrebaltieri

**Reply by andrebaltieri** [reply]
Boa tarde, @thyagojcosta, como vai?

É difícil falar pois ambas soluções podem funcionar, vai do que você tem mais conhecimento. Não rola fazer um teste com cada stack para ver qual se adapta melhor?

**Reply by thyagojcosta** [reply]
Valeu! Tenho mais conhecimento com Electron, mas vou fazer o teste com as duas!
